### PR TITLE
RWA-3617: Set pgsql_storage_mb

### PR DESCRIPTION
**Pull request by SammiChong**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RWA-3617

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: infrastructure/main.tf [hunk 1]
```diff
@@ -47,13 +47,13 @@ module "wa_task_management_api_database_flexible" {
...
-  business_area      = var.business_area
-  env                = var.env
+  source        = "git@github.com:hmcts/terraform-module-postgresql-flexible?ref=master"
+  product       = var.product
+  component     = var.component
+  name          = "${var.postgres_db_component_name}-postgres-db-flexible"
+  location      = var.location
+  business_area = var.business_area
+  env           = var.env
   pgsql_databases = [
```
Comment (raghera) [inline infrastructure/main.tf:57]: seems to be just formatting changes here?  Can you undo those please?

File: infrastructure/main.tf [hunk 2]
```diff
@@ -83,21 +84,21 @@ module "wa_task_management_api_database_flexible_replica" {
...
   }
 
-  source             = "git@github.com:hmcts/terraform-module-postgresql-flexible?ref=master"
-  product            = var.product
-  component          = var.component
-  name               = "${var.postgres_db_component_name}-postgres-db-flexible-replica"
-  location           = var.location
-  business_area      = var.business_area
-  env                = var.env
+  source        = "git@github.com:hmcts/terraform-module-postgresql-flexible?ref=master"
```
Comment (raghera) [inline infrastructure/main.tf:87]: same here for formatting.

File: infrastructure/variables.tf [hunk 3]
```diff
@@ -49,6 +49,10 @@ variable "business_area" {
   default = "cft"
 }
 
-variable jenkins_AAD_objectId {}
+variable "pgsql_storage_mb" {
+  default = 65536
+}
```
Comment (raghera) [inline infrastructure/variables.tf:54]: This could change it everywhere?  Were we advised to do that?

File: infrastructure/main.tf (hunk 1 above)
Comment (SammiChong) [reply infrastructure/main.tf:57]: Looks like the bot automatically formatted it this way. https://github.com/hmcts/wa-task-management-api/pull/1279/commits/856534032cc781fb54d3a7cb5f1784fe08efce2c

File: infrastructure/variables.tf (hunk 3 above)
Comment (SammiChong) [reply infrastructure/variables.tf:54]: This is the default size https://github.com/hmcts/terraform-module-postgresql-flexible?tab=readme-ov-file#input_pgsql_storage_mb so I'm defining it here and then setting the size it got upgraded to in Azure in demo.tfvars

Comment (raghera) [reply infrastructure/variables.tf:54]: The default should be set already in the terraform module so we shouldn't need to set it ourselves (you can prob find it in the module repo).  You should therefore just be able to override in demo where we want to change it?

Hopefully that will avoid all the destroy labels in prod etc.

File: infrastructure/main.tf (hunk 1 above)
Comment (raghera) [reply infrastructure/main.tf:57]: That's annoying it's made it worse

Comment (raghera) [reply infrastructure/main.tf:57]: Can leave as is in that case.

File: infrastructure/variables.tf (hunk 3 above)
Comment (SammiChong) [reply infrastructure/variables.tf:54]: When I tried to remove the default form the file I got a this error, "Error: Reference to undeclared input variable", looks like I'll have to have it set.

**Review by raghera — DISMISSED** [review]
Approved but please co-ordinate deployment since there's some performance changes we want to get green first.

**Review by raghera — APPROVED** [review]

**Pull request merged**